config/jobs/periodic/containerd/test-containerd-periodics.yaml

```diff
@@ -0,0 +1,47 @@
+periodics:
+  - name: periodic-containerd-test-ppc64le
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    interval: 24h
+    extra_refs:
+      - base_ref: main
+        org: containerd
+        repo: containerd
+        workdir: true
+    spec:
+      nodeSelector:
+        dedicated: runtimesTeam
+      tolerations:
+      - key: dedicated
+        operator: "Equal"
+        value: runtimesTeam
+        effect: "NoSchedule"
+      containers:
+      - image: quay.io/powercloud/docker-ce-build
+        command:
+          - /bin/bash
+        args:
+          - -c
+          - |
+            set -o errexit
+            set -o nounset
+            set -o pipefail
+            set -o xtrace
+            # Start docker
+            wget -O dockerctl.sh https://raw.githubusercontent.com/ppc64le-cloud/docker-ce-build/main/dockerctl.sh
+            chmod +x dockerctl.sh
+            ./dockerctl.sh start
+
+            echo "Build docker image to test containerd within a container"
+            cd contrib
+            docker build -t containerd-test -f Dockerfile.test ../
+            echo "Run docker image"
+            docker run --privileged -v /tmp:/tmp/docker_tmp -v ${ARTIFACTS}:${ARTIFACTS} --tmpfs /var/lib/containerd-test containerd-test bash -c "script/setup/install-gotestsum; GOTEST='gotestsum --' GOTESTSUM_JUNITFILE='junit_test.xml' make binaries install test; cp junit_test.xml ${ARTIFACTS}"
+            cd ..
+
+            # Stop docker
+            ./dockerctl.sh stop
```

Comment:
@EGuesnet The job ran successfully on the new ppc build cluster scheduling the pod to the dedicated worker node.
https://prow.ppc64le-cloud.org/view/s3/prow-logs/logs/periodic-containerd-test-ppc64le/1501897928216678400.

I have a minor question on why process ID 18 is printed at the end of the log above job. Just curious to know.
I have gone through your [dockerctl.sh](https://raw.githubusercontent.com/ppc64le-cloud/docker-ce-build/main/dockerctl.sh) but couldnt figure it out.

Otherwise, it looks good to me.

Reply:
The ID is printed by `while /usr/bin/pgrep ${DAEMON}`. I am able to reproduce with simple example
```
sleep 20 &
while pgrep sleep
do
sleep 5
done
```